src/mesh_motion/MotionRotation.C

```diff
@@ -0,0 +1,200 @@
+
```

Comment:
Maybe I'm missing something with the rebase, but don't these files already exist?